.circleci/config.yml

```diff
@@ -813,7 +813,7 @@ jobs:
       - chown_system_paths
       - run:
           name: Install Documentation Build Dependencies
-          command: python3 -m pip install --user .[docs]
+          command: python3 -m pip install -r docs-requirements.txt
```

Comment:
Might still need `--user` (?) - don't know if this is influencing the build or not, but this is what we had before the `docs` extra - https://github.com/nucypher/nucypher/commit/5ad41dad1342409b16c69ec835445df295ab47c9#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L814.